app/views/spree/shared/_user_form.html.erb

```diff
@@ -24,7 +24,7 @@
    <div class="password-hint">Minimum 7 characters. Must include at least 1 letter and 1 number.</div>
 </div>
 <div  class="form-group form-floating has-icon mb-20">
-   <%= f.password_field :password, :class => 'form-control password-control', :placeholder => Spree.t(:confirm_password) %>
+   <%= f.password_field :password_confirmation, :class => 'form-control password-control' %>
```

Comment:
Please add placeholder to this field: , :placeholder => Spree.t(:confirm_password)